scripts/iql-lesson-sync/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[build-system]
+requires = ["setuptools>=61.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "iql-lesson-sync"
+version = "0.0.1"
+authors = [
+  { name="Frank Harkins", email="frankharkins@hotmail.co.uk" },
```

Comment:
Our convention is to use the Qiskit team:

https://github.com/Qiskit/qiskit_sphinx_theme/blob/086486dedd9c19e8ee31bff6f8c6b9fc611be6c5/pyproject.toml#L12

I'd be fine with having both authors though!